extensions/wikia/RTE/RTE.class.php

```diff
@@ -61,6 +61,37 @@ public static function replacePlaceholder($var) {
 		$data = RTEData::get('placeholder', intval($var[1]));
 
 		if($data) {
+			wfProfileIn(__METHOD__);
+
+			// XW-2433 - hack starts
+			if ($data['type'] == 'double-brackets' || $data['type'] == 'ext') {
+				global $wgTitle;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)